packages/framework/block-std/src/gfx/controller.ts

```diff
@@ -20,21 +23,27 @@ import {
 } from './surface/element-model.js';
 import { Viewport } from './viewport.js';
 
-export class GfxController {
+export class GfxController extends LifeCycleWatcher {
   private _disposables: DisposableGroup = new DisposableGroup();
 
   private _surface: SurfaceBlockModel | null = null;
 
+  static override key = 'gfxController';
+
   readonly grid: GridManager;
 
   readonly layer: LayerManager;
 
-  std: BlockStdScope;
+  override std: BlockStdScope;
```

Comment:
I think you can safely remove this line because you can get `this.std` in `LifeCycleWatcher`.